sources/packages/backend/apps/queue-consumers/src/processors/schedulers/base-scheduler.ts

```diff
@@ -48,6 +48,20 @@ export abstract class BaseScheduler<T> implements OnApplicationBootstrap {
    * any old cron job delete it and add the new job to the queue.
    */
   async onApplicationBootstrap(): Promise<void> {
+    // Stops this scheduler from running if the isActive status is false
+    // and it falls into the scheduler-queues category (cron returns a value).
+    const queueConfigurations =
```

Comment:
Can we use the same method used in the mait.ts also here (`queueService.queueConfigurationModel`)?